text_mining.py

```diff
@@ -0,0 +1,140 @@
+"""
+This file will retrieve the tweets of 4 French presidential candidates and return a graph of the 
+sentiment analysis done on the candidates
+
+Author: Emily Lepert
+"""
+
+import twitter
+import numpy as np
+import math
+from unidecode import unidecode
+from textblob import TextBlob
+from textblob_fr import PatternTagger, PatternAnalyzer
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import matplotlib as mpl
+
+def retrieve_text(name, number):
+	consumer_k = open('consumer_key.txt').read().strip()
+	consumer_s = open('consumer_secret.txt').read().strip()
+	access_key = open('access_token_key.txt').read().strip()
+	access_secret = open('access_token_secret.txt').read().strip()
```

Comment:
If you just create a python file with those API keys as variables, you can just import that module and directly reference those API keys as variables.